rasgotransforms/rasgotransforms/column_transforms/math/math.yaml

```diff
@@ -18,4 +18,18 @@ example_code: |
       math_ops=['SALESAMOUNT * 10', 'SALESAMOUNT - TAXAMT'],
       names=['Sales10', 'SalesNet'])
 
-  ds2.preview()
\ No newline at end of file
+  ds2.preview()
+dataset-id: 74
+output-cols:
+  - SALESAMOUNT
+  - TAXAMT
+  - DUEDATE
+transform-details:
+  transform-name: math
```

Comment:
I don't think we need the transform name here, since we can get the value from the `file_name` when looping through the transforms while generating docs.

would be better to remove it so ppl don;lt need to type extra info for each transform YAML.